OpenRobertaServer/staticResources/js/app/roberta/controller/galleryList.controller.js

```diff
@@ -18,14 +18,47 @@ define([ 'require', 'exports', 'log', 'util', 'comm', 'message', 'guiState.contr
      */
     function init() {
 
+        initGalleryToolbar();
         initGalleryList();
         initGalleryListEvents();
         LOG.info('init gallery list view');
     }
     exports.init = init;
 
+    function getRobotGroups(){
+        const robots = GUISTATE_C.getRobots();
+        const groups = {};
+
+        const coerceName = (name, group) => {
```

Comment:
Unfortunately we cannot use ECMAScript 6 features in our code because of backwards compatibility. Please rewrite this.